x-pack/filebeat/module/panw/panos/config/input.yml

```diff
@@ -215,6 +215,11 @@ processors:
         observer.hostname: 33
         panw.panos.sequence_number: 34
         panw.panos.actionflags: 35
+        panw.panos.selection_type: 37
+        panw.panos.response_time: 38
+        panw.panos.priority: 39
+        panw.panos.attempted_gateways: 40
+        panw.panos.gateway: 41
```

Comment:
So it's also extracting more fields for GLOBALPROTECT. Out of curiosity, are these documented? I don't see them in the docs.

I was worried that this could cause a failure if the array has less entries, but realized that the `fail_on_error: false` above will make this backwards compatible.

Reply:
I have to find where I found them because I didn't see them originally when I did the last PR but then I saw them

Reply:
Found it https://docs.paloaltonetworks.com/pan-os/10-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/globalprotect-log-fields.html